### PR TITLE
Update release notes for v0.15.1

### PR DESCRIPTION
**Pull request by leandro-lucarella-frequenz**
Signed-off-by: Leandro Lucarella <leandro.lucarella@frequenz.com>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,16 +2,12 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+This is a bugfix release that mainly only uses more up to date dependencies and extend the range of supported dependencies.  There is technically one breaking change though, but this is hardly used by anyone.
```
Comment (daniel-zullo-frequenz) [inline RELEASE_NOTES.md]: I think here should be either `mainly` or `only` but not both

Comment (daniel-zullo-frequenz) [reply RELEASE_NOTES.md]: s/extend/extends

Comment (leandro-lucarella-frequenz) [reply RELEASE_NOTES.md]: Fixed.

**Review by daniel-zullo-frequenz — COMMENTED** [review]
Just a cosmetic to check for and LGTM

File: RELEASE_NOTES.md (hunk 1 above)
Comment (daniel-zullo-frequenz) [reply RELEASE_NOTES.md]: sorry, my previous comment about `s/extend/extends` was missed. do you want to update it? Anyway I'm happy  to approve it as it is

**Review by daniel-zullo-frequenz — APPROVED** [review]

**Pull request merged**